src/components/NavBar.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { Fragment } from "react";
+import { Link } from "react-router-dom";
+
+export default function NavBar()
+{
+    return (
+        <Fragment>
```

Comment:
As I see, in the MainLayout.tsx you used a short Fragment syntax <></>, here you used a different syntax. Please, choose one syntax and use it across the app. We prefer using the short syntax <></>